### PR TITLE
Updated the links in the readme buttons

### PR DESCRIPTION
**Pull request by rohit1817**
This is my first ever contribution on GitHub. I've updated all the links of the buttons as requested. If I made any mistakes, please let me know so I can learn and improve. Thank you for this opportunity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # IITM-BS Bots for staying up to date with all the events and deadlines
 
 Welcome to the IIT Madras-BS Uodates Bot repository ! This repository contains two folders: one for a WhatsApp bot and another for a Telegram bot. These bots are designed to provide details about today's live sessions and schedules specifically tailored for BS students  
-![Discord](https://img.shields.io/discord/1173095754961801286)
-![GitHub issues](https://img.shields.io/github/issues/shubhamatkal/IITM-BS-bots)
-![GitHub forks](https://img.shields.io/github/forks/shubhamatkal/IITM-BS-bots)
-![GitHub Repo stars](https://img.shields.io/github/stars/shubhamatkal/IITM-BS-bots)
-![GitHub contributors](https://img.shields.io/github/contributors/shubhamatkal/IITM-BS-bots)
+![Discord](https://discord.gg/hux9vWjVhR)
```
Comment (shubhamatkal) [inline README.md:4]: image
when i added this line of code in readme , it is not loading the image of button , actually the thing is we want to have that button , but also in that we want the link , i think you should refer [image shield web page](https://shields.io/) for more info on how to get that , Pls revert once u work on that , i hope that you will work on this issue , if any more help needed , kindly join discord server , and also dont forget to star the repo , thanks

**Review by shubhamatkal — COMMENTED** [review]
image

when i added this line of code in readme , it is not loading the image of button , actually the thing is we want to have that button , but also in that we want the link , i think you should refer [image shield web page](https://shields.io/) for more info on how to get that , Pls revert once u work on that , i hope that you will work on this issue , if any more help needed , kindly join discord server , and also dont forget to star the repo , thanks

File: README.md (hunk 1 above)
Comment (rohit1817) [reply README.md:4]: Oh looks like I misunderstood the issue. Thanks for explaining. I will try to work on it.

**Review by rohit1817 — COMMENTED** [review]
I added the links to the images. It should work this time as described.

**Review by shubhamatkal — APPROVED** [review]
great thanks, see u in server :)

**Comment by shubhamatkal** [comment]
@rohit1817  can you pls work on this [issue](https://github.com/shubhamatkal/IITM-BS-bots/issues/5)

**Pull request merged**